shared/actions/profile/proofs.js

```diff
@@ -110,6 +110,7 @@ function* _addServiceProof(service: ProvablePlatformsType): Saga.SagaGenerator<a
       promptPosted: false,
       service,
       username: '',
+      sigVersion: 0, // Default sigVersion will be used.
```

Comment:
fyi you can also just make this optional so we don't have to actually pass it in from the front end